dist/profile/manifests/jiraveg.pp

```diff
@@ -0,0 +1,32 @@
+# profile for jira
+class profile::jiraveg (
+  $jira_javahome,
+  $jira_version
+){
+  $pkgs = ['openjdk-7-jre-headless', 'facter']
```

Comment:
Interesting... i will double check. facter package must install and own shared directory either this will cause problems for FS and package manager consistency